tests/std/tests/P0009R18_mdspan_layout_left/test.cpp

```diff
@@ -0,0 +1,370 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <array>
+#include <cassert>
+#include <concepts>
+#include <cstddef>
+#include <mdspan>
+#include <span>
+#include <type_traits>
+
+#include "test_mdspan_support.hpp"
+
+template <class Mapping, class... Indices>
+concept CanInvokeCallOperatorOfMapping = requires(Mapping mapping, Indices... i) {
+                                             { mapping(i...) } -> same_as<typename Mapping::index_type>;

```

Comment:
Similar comment to the other test, it would be nice to avoid the `mapping` name here, even though it is not shadowing. Occurs below, and there's a variable named `copy` that raises my eyebrows.